config/application.rb

```diff
@@ -66,8 +66,7 @@ class Application < Rails::Application
         end
 
         ping.check :elasticsearch do
-          Chewy.client.cluster.health["status"] == "green"
-          'ok'
+          Chewy.client.cluster.health['status'] == 'green' ? 'ok' : false  
```

Comment:
Looks like Pinglish ignores the message of the exception, it just returns the same message for `false` and `raise`.

I agree that this `?:` expression is really confusing, how about we just returned the `status` instead?